common/app/common/commercial/BrandHunter.scala

```diff
@@ -6,29 +6,25 @@ import org.joda.time.DateTime
 
 object BrandHunter {
 
-  private def findBranding(activeBrandings: Option[Seq[Branding]],
-                           publicationDate: Option[DateTime],
-                           edition: Edition): Option[Branding] = {
+  def findBranding(
+    activeBrandings: Option[Seq[Branding]],
+    edition: Edition,
+    publicationDate: Option[DateTime]
+  ): Option[Branding] = {
     activeBrandings flatMap (_ find (_.isTargeting(publicationDate, edition)))
   }
 
-  def findSectionBranding(section: Section, publicationDate: Option[DateTime], edition: Edition): Option[Branding] = {
-    findBranding(section.activeBrandings, publicationDate, edition)
-  }
+  def findContentBranding(content: ContentType, edition: Edition): Option[Branding] = {
+    val publicationDate = Some(content.trail.webPublicationDate)
 
-  def findTagBranding(tag: Tag, publicationDate: Option[DateTime], edition: Edition): Option[Branding] = {
-    findBranding(tag.properties.activeBrandings, publicationDate, edition)
-  }
+    val brandingBySection = content.metadata.section.flatMap { section =>
+      findBranding(section.activeBrandings, edition, publicationDate)
+    }
 
-  def findFrontBranding(frontProps: FrontProperties, edition: Edition): Option[Branding] = {
-    findBranding(frontProps.activeBrandings, publicationDate = None, edition)
-  }
+    val brandingByTags = content.tags.tags.flatMap { tag =>
```

Comment:
The frontend content model goes through occasional clean ups but there's a lot of scope for more work.  It would be less confusing if it followed the capi result more closely.